integration/tracing/tracing-opentelemetry/src/main/java/org/apache/cxf/tracing/opentelemetry/OpenTelemetryStartInterceptor.java

```diff
@@ -51,5 +56,19 @@ public void handleMessage(Message message) throws Fault {
         if (holder != null) {
             message.getExchange().put(TRACE_SPAN, holder);
         }
+
+        HttpServletRequest request = (HttpServletRequest)message.getContextualProperty("HTTP.REQUEST");
+        if (request != null) {
+            Span.current().setAttribute(ClientAttributes.CLIENT_ADDRESS, request.getRemoteAddr());
```

Comment:
Please use `holder.getScope().getSpan()`, there is no need to use `Span.current()` - we know the exact span